lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -61,6 +61,12 @@ def execute(command, options = {})
         command_runner.output
       end
 
+      def response_on_execute(command, options = {})
+        result_msg = execute(command, options)
+        result = $CHILD_STATUS.success?
```

Comment:
Having the exit code, rather than true/false might be more universal, especially if we cared about the exit code